DEPLOYMENT_GUIDE.md

```diff
@@ -1,13 +1,19 @@
 # Deployment Guide
 
-This guide walks through the process of deploying contracts to public Ethereum networks and verifying contracts on Etherscan. The process requires an Ethereum mnemonic phrase, Etherscan API key, and Infura API key. All of these are free to get. All commands should be run out of the root folder of the contract you want to deploy. For example, to deploy Types.sol, run commands within the `source/types` package.
+This guide walks through the process of deploying contracts to public Ethereum networks and verifying contracts on Etherscan. The process requires an Ethereum mnemonic phrase, Etherscan API key, and Infura API key. All of these are free to get. All commands should be run out of the `utils/deployer` folder. 
 
```

Comment:
If you're deploying to development, there are a few pre-steps that need to take place. Open the `utils/deployer/migrations/1_initial_migration.js`, and fill in the sections that say "FILL_WITH_ERC20". These variables are expecting contract addresses that map to two different contract addresses. The STAKING_TOKEN_ADDRESS can be any standard ERC20 contract. the WETH_ADDRESS expects a contract that can take in ETH transform the value into a ERC20 token. Samples of both these contracts can be found within @airswap/tokens.